api4/post.go

```diff
@@ -396,6 +397,54 @@ func getPost(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getPostsByIds(c *Context, w http.ResponseWriter, r *http.Request) {
+	var postIDs []string
+
+	json.NewDecoder(r.Body).Decode(&postIDs)
+	if postIDs == nil {
+		c.SetInvalidParam("post_ids")
+		return
+	}
+
+	var posts = []*model.Post{}
+
+	for _, postID := range postIDs {
+		post, err := c.App.GetSinglePost(postID)
```

Comment:
Should we maybe get all the posts from the DB in a single query? The `getUsersByIds` one works like that